molecule/builder/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+"""
+Import variables from vars.yml and inject into pytest namespace
+"""
+
+import os
+import yaml
+
+
+def pytest_namespace():
+    """ Return dict of vars imported as 'securedrop_test_vars' into pytest
+        global namespace
+    """
+    filepath = os.path.join(os.path.dirname(__file__), "vars.yml")
+    with open(filepath, 'r') as f:
+        return dict(securedrop_test_vars=yaml.safe_load(f))
```

Comment:
Hmm, we'll need to duplicate these conftest.py files across multiple scenarios in Molecule (has bearing on #2218). I wonder if there's a better way?

Reply:
Well FYI , #2218 doesnt need this because I was unable to use molecule to kick off the testinfra tests there in a way that wouldn't cause a huge refactor. In that PR, we are calling the testinfra tests with a similar shell script that we currently have (took existing and stripped away unneeded bits)